build/Dockerfile.local

```diff
@@ -0,0 +1,57 @@
+## Copyright 2018 The Nakama Authors
```

Comment:
Do we need this file?

Reply:
This is sometimes useful for development to build a Nakama container based on the local changes instead of pulling from the repository, but I can remove it if deemed unnecessary